app/code/Magento/InventoryIndexer/Model/StockCondition/GetManageStockCondition.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Model\StockCondition;
+
+use Magento\CatalogInventory\Model\Configuration;
+
+/**
+ * Condition for manage_stock configuration.
+ */
+class GetManageStockCondition implements GetStockConditionInterface
```

Comment:
GetManageStockCondition and  GetStockConditionInterface
should not belong to InventoryIndexer module 
from modularity perspectives